tests/flow/test_order_by.py

```diff
@@ -0,0 +1,33 @@
+from redisgraph import Graph
+from base import FlowTestsBase
+
+GRAPH_ID = "order_by_test"
+redis_graph = None
+
+
+class testOrderBy(FlowTestsBase):
+    def __init__(self):
+        super(testOrderBy, self).__init__()
+        global redis_graph
+        redis_con = self.env.getConnection()
+        redis_graph = Graph(GRAPH_ID, redis_con)
+
+    def test_multiple_order_by(self):
+        q = """CREATE (:Person { id: 622, name: "Mo" })"""
+        redis_graph.query(q)
+
+        q = """CREATE (:Person { id: 819, name: "Bing" })"""
+        redis_graph.query(q)
+
+        q = """CREATE (:Person { id: 819, name: "Qiu" })"""
+        redis_graph.query(q)
```

Comment:
Please see `test_bound_variables.py` `populate_graph` function on how to populate a graph.